client/jetpack-connect/plans-landing.jsx

```diff
@@ -86,6 +90,50 @@ class PlansLanding extends Component {
 		this.props.recordTracksEvent( 'calypso_jpc_help_link_click' );
 	};
 
+	handleInfoButtonClick = info => {
+		this.props.recordTracksEvent( 'calypso_jpc_help_' + info + '_click' );
+	};
+
+	plansExtendedInfo = () => {
+		const { translate } = this.props;
+		const headerText = translate( 'Unsure which plan is right for you?' );
+		const subheaderText = translate(
+			'Read out guide to choosing the right plan or compare plans in more details.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Read our guide to choosing the right plan or compare plans in more detail.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).